src/adapter/src/coord/timeline.rs

```diff
@@ -95,6 +98,13 @@ impl<T: TimestampManipulation> TimestampOracle<T> {
         if next.less_equal(&self.state.write_ts) {
             next = self.state.write_ts.step_forward();
         }
+        // Cap how much we advance the write timestamp by for any single increment. This is to
+        // help protect against scenarios where a clock may jump forward by a large amount and
+        // then jump backwards.
+        next = std::cmp::min(
```

Comment:
We should log when this happens. How rarely do we expect this to trigger? If "pretty rarely" then maybe `error!` (because it'll go to sentry), otherwise `warn!`?

This should allow us to understand if we ever get into a case where the system clock jumps forward by a lot and this code is preventing the oracle from catching up (which we assume won't happen because we have well configured clocks with chrony, but this will help verify that assumption).

Reply:
I added a log, I think this would be a rare scenario so I made it `error!`.